ginga/colors.py

```diff
@@ -781,7 +781,7 @@ def lookup_color(name, format='tuple'):
 def resolve_color(color):
     if isinstance(color, str):
         r, g, b = lookup_color(color)
-    elif isinstance(color, collections.Sequence):
+    elif isinstance(color, collections.abc.Sequence):
```

Comment:
This emits deprecation warning in Python 3.8. Even though we currently have a setting under `ginga/conftest.py` to turn all deprecation warnings to errors, it is not happening on Travis. I moved `conftest.py` to root dir here and it seems to work again.

Reply:
Ah, wait, maybe Travis never used Python 3.8 in the first place! No matter, `conftest.py` should go to root dir anyway.